include/SDL3/SDL_video.h

```diff
@@ -200,7 +200,7 @@ typedef enum SDL_FlashOperation
  *
  * \sa SDL_GL_CreateContext
  */
-typedef void *SDL_GLContext;
+typedef struct struct_SDL_GLContext *SDL_GLContext;
```

Comment:
```suggestion
typedef struct SDL_GLContext *SDL_GLContext;
```

Reply:
For clarity, I added the `struct_` prefix to differentiate against the `typedef SDL_Window SDL_Window` (which has no `*`).

Reply:
Good point.

Reply:
Looks like llvm's type checker does not allow `typedef struct SDL_GLContext *SDL_GLContext`.
Perhaps you have a better suggestion then a `struct_` prefix?

Reply:
Alternatively, we make the typedef `typedef struct SDL_GLContext SDL_GLContext` and modify all functions returning and accepting a `SDL_GLContext` struct to accept a `SDL_GLContext *` pointer instead of `SDL_GLContext`.

Reply:
`typedef struct SDL_GLContextObject *SDL_GLContext;`?

Reply:
> `typedef struct SDL_GLContextState *SDL_GLContext;`?

That will work with the least amount of work :)